gradle/libs.versions.toml

```
@@ -93,7 +93,6 @@ vavr = { group = "io.vavr", name = "vavr", version = "0.10.4" }
 apache-commons-lang = { group = "org.apache.commons", name = "commons-lang3", version = "3.14.0" }
 apache-commons-io = { group = "commons-io", name = "commons-io", version = "2.16.1" }
 apache-commons-collections = { group = "org.apache.commons", name = "commons-collections4", version = "4.4" }
-trove = { group = "trove", name = "trove", version = "1.0.2"}
 trove4j = { group = "net.sf.trove4j", name = "trove4j", version = "3.0.3" }
```

```suggestion
trove4j = { group = "net.sf.trove4j", name = "core", version = "3.1.0" }
```
In the screenshot it says this got moved to "core". There's also a newer version there. Should we use this instead?

---

sure, if modelix compiles.

---

fixed in 9e937bf6e7db7dbd7cceaa1496a238f97e0d3f10